op-chain-ops/deployer/deployer.go

```diff
@@ -89,6 +98,21 @@ func NewBackendWithGenesisTimestamp(ts uint64, shanghai bool) *backends.Simulate
 		chainConfig.ShanghaiTime = u64ptr(0)
 	}
 
+	var alloc core.GenesisAlloc = core.GenesisAlloc{
```

Comment:
```suggestion
     alloc := core.GenesisAlloc{
```

small nit, var isn't needed here